src/lib/reasoners/satml_frontend.ml

```diff
@@ -839,23 +852,24 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
         (*SAT.update_lazy_cnf activate ~dec_lvl;*)
         SAT.assume env.satml unit nunit f ~cnumber:0 activate ~dec_lvl;
       with
-      | Satml.Unsat (lc)  -> raise (IUnsat (env, make_explanation lc))
+      | Satml.Unsat (lc) -> raise (IUnsat (env, make_explanation lc))
       | Satml.Sat -> assert false
 
-
-  let assume_aux_bis ~dec_lvl env l =
+  let pre_assume_aux env l =
```

Comment:
introduction of function `pre_assume_aux` is probably not needed if we don't use it (anymore) in `push`